src/domain/individualCustomer/boatsCard/BoatsCard.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import Paragraph from '../../../common/paragraph/Paragraph';
+import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
+import styles from './boatsCard.module.scss';
+
+interface Boat {
+  id: string;
+  boatType: string;
+  registrationNumber: string;
+  boatWidth: string;
+  boatLength: string;
+  boatDepth: string;
+  boatWeight: string;
+  boatName: string;
+  boatBrand: string;
+}
+
+interface LargeBoat extends Boat {
+  boatPower: string;
+  boatMaterial: string;
+  purpose: string;
+  inspection: string;
+  insurance: string;
+}
+
+export interface BoatsCardProps {
+  boats: (Boat | LargeBoat)[];
+}
+
+const BoatsCard: React.SFC<BoatsCardProps> = ({ boats }) => {
+  const { t } = useTranslation();
+  const isLargeBoat = (boat: LargeBoat | Boat): boat is LargeBoat =>
+    (boat as LargeBoat).boatPower !== undefined;
+
+  return (
+    <div>
```

Comment:
I think this `div` should be a `Card` and the nested `Card` under this should be a `Paragraph`.